CONTRIBUTING.md

```diff
@@ -39,11 +39,12 @@ Finally, submit a pull request, with the following rules:
 2. Do include relevant issue numbers in the description
 3. Include a concise, specific description of the complete pull request in the description
 4. Take care to exclude project workspace files and other collateral unnecessary to this project 
+5. Think about adding screenshot (either complete or partial) for any change which has a visual impact.
```

Comment:
```suggestion
5. Add a screenshot (either complete or partial) for any change which has a visual impact.
```

Reply:
^ Correcting both grammar and making it more forceful - we've made it a norm to include screenshots in PRs with a visual impact.